exonum-dictionary.txt

```diff
@@ -86,8 +86,10 @@ postvote
 precheck
 precommit
 precommits
+precommiters
```

Comment:
&lt;span class="grammarnazi"&gt;It is precommitters, with double t.&lt;/span&gt;